contrib/babelfishpg_tsql/src/pltsql_utils.c

```diff
@@ -1718,6 +1719,14 @@ Oid get_sys_varcharoid(void)
 	return sys_varcharoid;
 }
 
+Oid get_sysadmin_oid(void)
+{
+	if (!OidIsValid(sysadmin_oid))
+		sysadmin_oid = get_role_oid("sysadmin", true);
```

Comment:
One question, will sysadmin change during runtime ?

Reply:
Once created, OID will not change.